style.css

```diff
@@ -1,3 +1,1076 @@
-h1 {
-  color: red;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@media only screen and (max-width: 768px) {
+  .main {
+    background-image: url("./image/body.png");
+    background-size: 100%;
+    background-position-y: 3rem;
+    height: auto;
+    background-repeat: no-repeat;
+  }
```

Comment:
- [ ] Even though you did an amazing job designing the Desktop version 🙌🏽, however, you also need to consider the mobile version to keep the page responsive. The current **Media Query** you have in your **style.css** will only apply to screen sizes of size `768px` and larger, this means that your styles will only apply to larger screen sizes, and that's why your page doesn't look as expected on mobile screens 🤔. Please consider adding the styles for your mobile screen sizes 👇🏽

![image](https://user-images.githubusercontent.com/43172164/208080744-647f59da-74f8-421d-8762-321fe87f2217.png)
![image](https://user-images.githubusercontent.com/43172164/208080963-48e1ff29-f1f7-4b98-8ca3-ae5eb9d80f86.png)
![image](https://user-images.githubusercontent.com/43172164/208081007-fe1aaf57-437a-4f59-8070-89802673eb82.png)
![image](https://user-images.githubusercontent.com/43172164/208081046-8e3c7a95-61a6-43d3-b1c7-e4aaf784c63a.png)

Reply:
I have decided that the hole project need change so I have changed everything even the structure and the classes